### PR TITLE
arshad-terrform

### PR DESCRIPTION
**Pull request by sulaim09**
terraform command

### CODE REVIEW

**Comment by sulaim09** [comment]
sir  plz check

**Review by AtiwadkarAjinkya — CHANGES_REQUESTED** [review]
Change the File format to .tf

File: arshad-Terrform.txt.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_instance" "web" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t3.micro"
```
Comment (AtiwadkarAjinkya) [inline arshad-Terrform.txt.txt]: Make it to t2.micro

**Review by AtiwadkarAjinkya — CHANGES_REQUESTED** [review]

**Review by sulaim09 — COMMENTED** [review]
changes file extension

**Pull request merged**